greeclimate/device.py

```diff
@@ -105,7 +105,11 @@ def generate_temperature_record(temp_f):
 TEMP_OFFSET = 40
 TEMP_MIN_F = 46
 TEMP_MAX_F = 86
-TEMP_TABLE = [generate_temperature_record(x) for x in range(TEMP_MIN_F, TEMP_MAX_F + 1)]
+TEMP_MIN_TABLE = 8
+TEMP_MAX_TABLE = 50
```

Comment:
I guess the idea here is that the device can report temperatures outside of the range, and need conversion, which is fine... but then minimum should also be reduced to give more headroom.